Sources/MessagingPush/PushHandling/PushClickHandler.swift

```diff
@@ -32,10 +31,9 @@ class PushClickHandlerImpl: PushClickHandler {
 
         // Now we are ready to handle the push click.
 
-        // Track metrics
-        if sdkConfig.autoTrackPushEvents {
```

Comment:
Thanks for the very thorough explanation Levi, I believe this `if` was added to avoid the automatic tracking, by removing it do we track metric even when sets this config as `false` ?

Reply:
The `if` statement has been moved out of the `PushClickHandler` so that it can be used for both manual and automatic push click handing. Whatever a customer chooses to do. 

[Here](https://github.com/customerio/customerio-ios/blob/b61b6d52b87f02ccb5f406bd559f7ff17e0d3411/Sources/MessagingPush/MessagingPush.swift#L69C1-L69C1) is where the `if` statement has been moved to. Does that answer your question?